dfget/config/config.go

```diff
@@ -196,8 +196,10 @@ type Config struct {
 	// Help show help information.
 	Help bool `json:"help,omitempty"`
 
-	// Client queue size.
-	// TODO: support setupFlags
+	// ClientQueueSize is the size of client queue
+	// which controls the number of pieces that can be processed simultaneously.
+	// It is only useful when the Pattern equals "source".
```

Comment:
I think that it's only useful when the pattern don't equal "source".

Reply:
Done.